### ISSUE TITLE
can we have <bolt> without <nut> ?

### ISSUE THREAD

**Issue by economidis-nick** [economidis-nick/createXSDforxMCF#6]
### Refers to question:

![nut_required_in_bolt](https://user-images.githubusercontent.com/33229181/54490162-3151b280-48bc-11e9-9a1c-18389d043cb4.PNG)

### Clarification needed:
I remember that, in one of the meetings, the **definition/difference of bolts vs screws** was:
> -- bolts do not drill their holes --  
> -- bolts can be replaced many times in the same hole 
> -- screw assemblies are not meant to be replacable/fixable with the same degree of fidelity 
> -- screws drill their own holes or create the holes' thread during screwing

Since the last part of a bolt could be the pre-machined hole in, say, the Engine Block, then this is not necessarily a screw, but a bolt.

**@DrCaFr : is the threaded connection in the above example considered a bolt, or a screw?**

If it is a bolt, then `<nut>` is **optional** as bolts not need them.

I think my claim, that the nut is optional has been captured by the example, in page 79:
![image](https://user-images.githubusercontent.com/33229181/54567544-df547e00-49dc-11e9-8de7-b1202f1fc820.png)

**Comment by DrCaFr** [comment]
Hi Nick,

sec. 7.5.1 starts with a pretty simple definition:

> differentiation between screws and bolts:
•	A screw has a tapped bore. 
•	A bolt needs a nut. 

I suggest sticking to this and fixing any later contradiction ;-)

Best regards,

  Carsten

**Comment by economidis-nick** [comment]
I agree.
So, what we're leaning towards is 

## Change 1: 
that this example, in page 79:
> ![image](https://user-images.githubusercontent.com/33229181/54567544-df547e00-49dc-11e9-8de7-b1202f1fc820.png)

should be changed into a **`<screw>`**

```xml
<connection_0d label="135"> 
    <threaded_connection diameter="10.0" length="50.0" 
                     head_diameter="16.0" head_height="5" thread_length=”35” 
                     torque=”80” angle=”30” pretension=”1200” >
        <normal_direction x="0" y="0" z="-10"/>
        <screw base=”3” />
    </threaded_connection>
    <loc> 1500.3809  838.75885  730.6529 </loc>
</connection_0d>
```

## Change 2:

for this:
> ![nut_required_in_bolt](https://user-images.githubusercontent.com/33229181/54490162-3151b280-48bc-11e9-9a1c-18389d043cb4.PNG)

we can set it back to  `<nut>`  **is** `required`,

--------


## Note on screw vs bolt definition

This definition, as simple as it appears, is somehow convoluted
> differentiation between screws and bolts:
> •	A screw has a tapped bore.
> •	A bolt needs a nut.


First of all, this sentence does not make any sense:
> •	A screw has a tapped bore.

"Bore" is a hole, and screws don't have holes.

Here's a dictionary definition:
> 1. the hollow part inside a gun barrel or other tube.
> 2. short for borehole.
> _synonyms: | borehole, hole, well, shaft, pit, passage, tunnel_


Even if we rephrase it to, say, 
> A screw **is inserted into** a tapped bore.

it still contradicts the `<flow_drilled>` definition, which is a `<screw>` 
that it taps its own hole (section 7.5.7.1)
> c. Screw thread tapping.

and it also fails to make any difference from a bolt, 
since `<nut>` has a tapped bore, but `<screw>` is not inserted in a nut -- only a `<bolt>` should.

I think this definition needs to be clarified by AK.

Perhaps we may only keep this sentence as a sole discriminator:
> •	A bolt needs a nut.

**Comment by DrCaFr** [comment]
I suggest rephrasing
> A screw has a tapped bore.
to
> A screw needs a tapped bore in a part it connects. Nuts are not parts in this sense. Tapped bores may be created by the screw itself during assembly. 

@economidis-nick: Do you feel more comfortable with this?

**Comment by economidis-nick** [comment]
The rule differentiating bolts and screws has been decided & defined by the AK25 group:
![image](https://user-images.githubusercontent.com/33229181/66331983-6139d380-e93c-11e9-91bb-ed2c7d706e2d.png)
* document will be updated according to the wiki definition
* examples in page 79 of document will _not_ be a screw 
* <nut/> will become optional because all bolts have a nut, by definition
Missing <nut/> implies that simply there is no information for the nut's dimensions
